package.json

```diff
@@ -38,6 +38,7 @@
     "graphql-upload": "8.1.0",
     "intersect": "1.0.1",
     "jsonwebtoken": "8.5.1",
+    "ldapjs": "^1.0.2",
```

Comment:
Can you pin the package? remove the `^`.

Reply:
done!